redbot/core/bank.py

```diff
@@ -400,19 +400,18 @@ def _invalid_amount(amount: int) -> bool:
 
     """
     if await is_global():
-        acc_data = (await _conf.user(member)()).copy()
-        default = _DEFAULT_USER.copy()
+        all_accounts = await _conf.all_users()
```

Comment:
Would it make more sense to use a context manager with a sentinel value default for account creation which would be impossible? It's not a required change, but it should simplify the new user case significantly.

Reply:
That's true. For now this works so I'm going to leave it as it is, however sentinel values are possible as defaults... You just need to be careful whether you specify a sentinel value as an empty dict of `None`.

When using a `NoneType` sentinel value, that value will not be treated as a `config.Group`, even if a dict later replaces it. Using an empty dict instead, the value will be treated as a `config.Group`, and programmers can utilise the extra methods of this object.